lib/policies/rate-limit/index.js

```diff
@@ -1,5 +1,15 @@
-'use strict';
-
 module.exports = {
-  policy: require('./rate-limit')
+  policy: require('./rate-limit'),
+  schema: {
+    type: 'object',
+    properties: {
+      rateLimitBy: {type: 'string'},
+      windowMs: {type: 'integer'},
```

Comment:
btw, can it be some description here or link to the doc? I'm ok with current version, just thinking how UI can show  help info

Reply:
Good point — JSON schema supports `title` and `description` property so we can definitely decorate those once we'll be able to forward this data to the UI.